pkg/controllers/provisioning/nodepool_test.go

```diff
@@ -1459,4 +1473,71 @@ var _ = Describe("NodePool/Provisioning", func() {
 			})
 		})
 	})
+	Context("Resource Calculations", func() {
+		It("should schedule based on the max resource requests of containers and initContainers with sidecarContainers for order type1", func() {
+			always := v1.ContainerRestartPolicyAlways
+
+			pod := test.Pod(test.PodOptions{
+				ResourceRequirements: v1.ResourceRequirements{
+					Limits:   v1.ResourceList{v1.ResourceCPU: resource.MustParse("2"), v1.ResourceMemory: resource.MustParse("1Gi")},
+					Requests: v1.ResourceList{v1.ResourceCPU: resource.MustParse("2")},
+				},
+
+				InitImage: "pause",
+
+				InitContainers: []test.InitContainersOptions{
+					{
+						RestartPolicy: &always,
```

Comment:
We could probably just keep the upstream constant value here

Reply:
sorry ? I dint get it

Reply:
I think he means you can inline it
```suggestion
						RestartPolicy: v1.ContainerRestartPolicyAlways,
```